vm/src/stdlib/winreg.rs

```diff
@@ -191,7 +191,8 @@ mod winreg {
         key.with_key(|k| k.get_raw_value(subkey))
             .map_err(|e| e.to_pyexception(vm))
             .and_then(|regval| {
-                let ty = regval.vtype as usize;
+                #[allow(clippy::redundant_clone)]
+                let ty = regval.vtype.clone() as usize;
```

Comment:
This smells like a bug somewhere else.

Reply:
seems like a clippy false positive tbh, I don't have windows so this was a pain to go through